### PR TITLE
Fix remarks, insert widoco

### PR DESCRIPTION
**Pull request by lecoqlibre**

### CODE REVIEW

File: specs/solid-indexing/index.html [hunk 1]
```diff
@@ -88,7 +130,219 @@ <h3>Namespaces</h3>
   </section>
 
   <section>
```
Comment (pchampin) [inline specs/solid-indexing/index.html:142]: some intro text at the top of the vocabulary section would be nice... Currently it is very abrupt.

File: specs/solid-indexing/index.html [hunk 2]
```diff
@@ -102,13 +356,16 @@ <h2>Indexes</h2>
...
       <h3>General indexes</h3>
 
-      <p>An index is a RDF document [[RDF11-CONCEPTS]] of type <code>idx:Index</code> containing entries which point to
-        instances conforming to a particular shape. The shape that the index or an entry is targeting is expressed with the
-        <code>idx:hasShape</code> predicate.
-      </p>
+      <p>An index is a RDF document [[RDF11-CONCEPTS]] of type <code>idx:Index</code>. It contains entries of type <code>idf:IndexEntry</code>, each one 
+        being linked to a shape with the <code>idx:hasShape</code> predicate.</p>
+        
+      <p>An entry MUST refer to either an instance or to a sub index with the respective predicates <code>idx:hasTarget</code> and <code>idx:hasSubIndex</code>.</p>
```
Comment (pchampin) [inline specs/solid-indexing/index.html]:
```suggestion
      <p>An entry MUST refer to either resources matching the shape  or to another index referring to such resources, with predicate <code>idx:hasTarget</code> or <code>idx:hasSubIndex</code> respectively.</p>
```

File: specs/solid-indexing/index.html [hunk 3]
```diff
@@ -44,7 +86,7 @@ <h3>Terminology</h3>
 
       <p>
         <dfn data-lt="indexes">Index</dfn>
-        A list of keywords with pointers to where further information about the keyword is found. An index summarizes or
+        A list of keyword with pointers to where further information about the keyword is found. An index summarizes or
         characterizes the content and its location.
```
Comment (pchampin) [inline specs/solid-indexing/index.html]: Why remove the plural??
Anyway, I didn't notice this definition before, but I don't find it satisfying. The notion of keyword does not match what we are doing (or only partially).

I can not make a suggestion here (this impacts non-modified lines), I will make a PR to the PR.

File: specs/solid-indexing/index.html (hunk 2 above)
Comment (pchampin) [reply specs/solid-indexing/index.html]: or, if you don't want to allow multiple values for hasTarget/hasSubIndex.

```suggestion
      <p>An entry MUST refer to a resource matching the shape  or to another index referring to such resources, with predicate <code>idx:hasTarget</code> or <code>idx:hasSubIndex</code> respectively.</p>
```

File: specs/solid-indexing/index.html [hunk 4]
```diff
@@ -102,13 +356,16 @@ <h2>Indexes</h2>
...
-        <code>idx:hasShape</code> predicate.
-      </p>
+      <p>An index is a RDF document [[RDF11-CONCEPTS]] of type <code>idx:Index</code>. It contains entries of type <code>idf:IndexEntry</code>, each one 
+        being linked to a shape with the <code>idx:hasShape</code> predicate.</p>
+        
+      <p>An entry MUST refer to either an instance or to a sub index with the respective predicates <code>idx:hasTarget</code> and <code>idx:hasSubIndex</code>.</p>
+
+      <p>An instance is...</p>
+
+      <p>A sub index is...</p>
```
Comment (pchampin) [inline specs/solid-indexing/index.html:378]:
```suggestion
```
No need for this with my proposed rewording above.

File: specs/solid-indexing/index.html [hunk 5]
```diff
@@ -102,13 +356,16 @@ <h2>Indexes</h2>
...
 
-      <p>An entry is of type <code>idx:IndexEntry</code> and targets a resource using the <code>idx:hasTarget</code>
-        predicate. The resource the entry is targeting MUST conform to the shape of the index.</p>
+      <p>A shape is...</p>
```
Comment (pchampin) [inline specs/solid-indexing/index.html:380]: suggest to copy the definition from SHACL (with a reference).

Maybe this is the place to explain that this spec recommends a very small subset of shapes.

Interestingly, we could write a SHACL shape of the SHALC shapes we allow :-)

**Review by pchampin — CHANGES_REQUESTED** [review]

File: specs/solid-indexing/index.html (hunk 3 above)
Comment (lecoqlibre) [reply specs/solid-indexing/index.html]: This definition was the one used in our different presentations (slides).

Comment (lecoqlibre) [reply specs/solid-indexing/index.html]: Fixed in https://github.com/Wimmics/solid-start/pull/47/commits/7c73164dd34d8aaddd7e8e64ca93e83bae74f4e2

File: specs/solid-indexing/index.html (hunk 1 above)
Comment (lecoqlibre) [reply specs/solid-indexing/index.html:142]: Fixed in https://github.com/Wimmics/solid-start/pull/47/commits/b71e281de8267a11f748a96e324bf6176435a126

**Pull request merged**